### DISCUSSION TITLE
Markdoc language server: nextjs partials

### DISCUSSION THREAD

**Discussion by amozoss** [markdoc/markdoc#439] [Q&A]
VScode is linking to s3-credentials.md, but it's getting an error when I click
```
{% partial file="s3-credentials.md" /%}
```
![Screenshot 2023-08-23 at 2 34 16 PM](https://github.com/markdoc/markdoc/assets/1147264/ff68b43d-6e31-4ac8-9bbd-4f3238840cbe)

My partials are in `src/markdoc` and they work out of the box with the next.js plugin.
```
src/markdoc
├── config.js
├── nodes.js
├── partials
│   └── s3-credentials.md
└── tags.js
```

here is my `markdoc.config.json`

```
[
  {
    "id": "storj-docs",
    "path": "app",
    "schema": {
      "path": "src/markdoc/config.mjs",
      "type": "esm",
      "property": "default",
      "watch": true
    }
  }
]
```
`config.mjs`
```
import tags from './tags'
import nodes from './nodes'

export default {
  tags,
  nodes,
}
```

Is there a way to specify the partials path for the Markdoc language server so the links work?

**Comment by rpaul-stripe** [comment]
The language server assumes that the partials are relative to the content root. We should add a way to make partial resolution configurable. This is also likely going to affect other features that make the same assumptions about where partials are located.
